### PR TITLE
Bugfix/language inheritance

### PR DESCRIPTION
**Pull request by bleto**
# Description

Bugfix for language tree and user language restrictions.

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [x] Unit Tests
- [x] e2e Test

# Checklist:

- [x] I have read the contribution requirements and fulfill them.
- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: modules/@ergo/core/components/TemplateGrid/TemplateGridItem.vue [hunk 1]
```diff
@@ -156,15 +152,14 @@ export default {
         }
 
         &__icon {
-            margin-right: 10px;
+            margin: 0 10px 0 -2px;
```
Comment (derpdead) [inline modules/@ergo/core/components/TemplateGrid/TemplateGridItem.vue:155]: Negative margin might cause unpredicted behaviour

File: modules/@ergo/core/components/TemplateGrid/TemplateGridItem.vue [hunk 2]
```diff
@@ -184,6 +179,7 @@ export default {
             flex: 0 1 auto;
             align-items: flex-start;
             opacity: 0;
+            margin-right: -10px;
```
Comment (derpdead) [inline modules/@ergo/core/components/TemplateGrid/TemplateGridItem.vue:182]: Same here, negative values for paddings and margins might produce artefacts

File: modules/@ergo/core/components/TemplateGrid/TemplateGridWrapper.vue [hunk 3]
```diff
@@ -213,7 +213,7 @@ export default {
             const children = this.filteredGridData.filter(e => e.parent === parent);
             const connectionHeight = this.rowHeight * (
                 row - (children.length ? children[0].row : 0) + 1
-            );
+            ) + 10;
```
Comment (derpdead) [inline modules/@ergo/core/components/TemplateGrid/TemplateGridWrapper.vue:216]: whats 10 value

Comment (bleto) [reply modules/@ergo/core/components/TemplateGrid/TemplateGridWrapper.vue:216]: it's an additional 10px to be sure that the line will connect to the previous element

**Pull request merged**